samples/05_content_publishers/pdf_table_to_pdf_map.ipynb

```diff
@@ -0,0 +1,1459 @@
+{
```

Comment:
<p>This is very useful and readers would like to reuse this. Could you add some comments explaining how new rows are added? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>